scripts/tests/prepare_tests.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+if [[ ! -f ".env" ]]
+then
+    echo ".env file does not exist"
+    touch .env
+    cat << 'EOF' > .env
+        export PATH=$PATH:~/Qt/6.2.4/macos/bin:$PATH
+        export QT_MACOS_BIN=~/Qt/6.2.4/macos/bin
+        MVPN_API_BASE_URL=http://localhost:5000
+        MVPN_BIN=build/tests/dummyvpn/dummyvpn
+        ARTIFACT_DIR=tests/artifact
+EOF
+fi
+
+# check if build directory exists
+if [[ ! -d "./build/tests/dummyvpn" ]]
+then
+    echo "dummyvpn dir does not exist, run cmake command to build it"
+    cmake -B build .
```

Comment:
I think the mocha beforeAll() function should check if the client exists and complain with a proper error message. Nothing more than this.

Reply:
will remove the build command and put that in a string text